python-sdk/nuscenes/predict/models.py

```diff
@@ -0,0 +1,169 @@
+# nuScenes dev-kit.
+# Code written by Freddy Boulton, Robert Beaudoin 2020.
+
+import abc
+from typing import Tuple
+import numpy as np
+from pyquaternion import Quaternion
+from nuscenes.predict import PredictHelper, convert_global_coords_to_local
+from nuscenes.eval.common.utils import quaternion_yaw
+from nuscenes.eval.predict.data_classes import Prediction
+
+KinematicsData = Tuple[float, float, float, float, float, float, float, float, float, float]
+
+def _kinematics_from_tokens(helper: PredictHelper, instance: str, sample: str) -> KinematicsData:
```

Comment:
Missing docstrings for arguments in this file.

Reply:
Added!